lib/ClangImporter/SwiftLookupTable.cpp

```diff
@@ -1793,21 +1797,35 @@ void importer::finalizeLookupTable(SwiftLookupTable &table,
   }
 }
 
+void SwiftLookupTableWriter::populateTableWithDecl(SwiftLookupTable &table,
+                                                   NameImporter &nameImporter,
+                                                   clang::Decl* decl) {
+  // Skip anything from an AST file.
+  if (decl->isFromASTFile())
+    return;
+
+  auto linkageDecl = dyn_cast<clang::LinkageSpecDecl>(decl);
+  if (linkageDecl) {
+    for (auto* decl : linkageDecl->noload_decls()) {
```

Comment:
`if (auto linkageDecl = ...)`
(If this is consistent with the style in the rest of the code.)